prov/efa/src/rxr/rxr_pkt_entry.c

```diff
@@ -363,17 +363,29 @@ ssize_t rxr_pkt_entry_sendmsg(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry
 
 ssize_t rxr_pkt_entry_sendv(struct rxr_ep *ep,
 			    struct rxr_pkt_entry *pkt_entry,
-			    fi_addr_t addr, const struct iovec *iov,
-			    void **desc, size_t count, uint64_t flags)
+			    fi_addr_t addr, uint64_t flags)
```

Comment:
pkt_entry has addr in it, so we do not need it in the argument.

Reply:
I thought about that, and felt that it might be intuitive to have the address as an argument if we want to send something to somewhere. 
And also tried to be consistent with the following definition of `rxr_pkt_entry_inject(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry, fi_addr_t addr)`

Reply:
but `pkt_entry` already has `addr` in it, I cannot think of a case that we want to send a packet entry to a different address.

If the current code is wrong, we should not follow it. We might want to add a refactor commit to fix the exiting one.

Also, since your are here, please add a doxygen type function header to the function.

Reply:
Fixed in new rev.